internal/observability/middleware_test.go

```diff
@@ -429,6 +438,7 @@ func TestMiddlewareSuccessSnapshot(t *testing.T) {
 	tags := metrics.Tags{
 		"dest":             "service",
 		"direction":        "inbound",
+		"transport":        "default",
```

Comment:
Let’s trace this and change it to "unknown", as you suggest.

Reply:
https://sourcegraph.com/github.com/yarpc/metrics@1e19de5b971489a45d178e12a0f72a78c70e300e/-/blob/tag.go#L34:2$references